examples/Makefile.am

```diff
@@ -25,8 +25,21 @@ EXAMPLE_CUDA_CFLAGS = $(CFLAGS_PEDANTIC)
 EXAMPLE_CUDA_CPPFLAGS =
 endif
 
+if HAVE_NVTX
+EXAMPLE_NVTX_LD_FLAGS = $(NVTX_LDFLAGS)
+EXAMPLE_NVTX_LIBS = $(NVTX_LIBS)
+EXAMPLE_NVTX_CFLAGS =
+EXAMPLE_NVTX_CPPFLAGS = $(NVTX_CPPFLAGS) -DHAVE_NVTX
+else
+EXAMPLE_NVTX_LD_FLAGS =
+EXAMPLE_NVTX_LIBS =
+EXAMPLE_NVTX_CFLAGS = $(CFLAGS_PEDANTIC)
+EXAMPLE_NVTX_CPPFLAGS =
+endif
+
 EXAMPLE_CCLD_FLAGS = -lucs -I$(includedir) -L$(libdir) -Wall -Werror -Wl,-rpath,$(libdir) \
-                     $(EXAMPLE_CUDA_LD_FLAGS) $(EXAMPLE_CUDA_LIBS) $(EXAMPLE_CUDA_CPPFLAGS)
+                     $(EXAMPLE_CUDA_LD_FLAGS) $(EXAMPLE_CUDA_LIBS) $(EXAMPLE_CUDA_CPPFLAGS) \
+                     $(EXAMPLE_NVTX_LD_FLAGS) $(EXAMPLE_NVTX_LIBS) $(EXAMPLE_NVTX_CPPFLAGS)
```

Comment:
why needed in hello world examples?

Reply:
This was a remnant from a previous commit that I missed taking out. Removed now.